react-native-libraries.json

```
@@ -9185,5 +9185,13 @@
     "ios": true,
     "android": true,
     "web": true
+  },
+  {
+    "githubUrl": "https://github.com/killserver/react-native-screenshot-prevent",
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false,
+    "dev": false
```

```suggestion
    "android": true
```

We can skip the falsy fields, besides that LGTM! 👍